daemon/graphdriver/overlayutils/overlayutils.go

```diff
@@ -13,6 +14,7 @@ func ErrDTypeNotSupported(driver, backingFs string) error {
 	if backingFs == "xfs" {
 		msg += " Reformat the filesystem with ftype=1 to enable d_type support."
 	}
-	msg += " Running without d_type support will no longer be supported in Docker 17.12."
-	return errors.New(msg)
+	msg += " Running without d_type support is not supported."
```

Comment:
Perhaps `a filesystem not implementing d_type is not supported`?

Reply:
Ah, yes, I can rephrase it a bit 👍